atlas/templates/Jenkinsfile.gotmpl

```diff
@@ -31,21 +37,24 @@ pipeline {
       steps {
         withDockerRegistry([credentialsId: "<insert-the-creds-id>", url: ""]) {
           dir("$DIRECTORY") {
-            sh "make docker push"
+            // The shell started by Jenkins should inherit the env vars defined above.
+            // Since Makefile.common uses ?= to set GIT_COMMIT and IMAGE_VERSION, then the values
+            // set above should override the defaults from Makefile.common.
+            sh "make docker"
```

Comment:
Even though signing is a push event, I would keep push here. We also need to support pushing to docker hub for the time being.

Reply:
OK - I was just trying to illustrate target state